runtime/vm/lookupmethod.c

```diff
@@ -193,6 +193,7 @@ processMethod(J9VMThread * currentThread, UDATA lookupOptions, J9Method * method
 					} else {
 							*exception = J9VMCONSTANTPOOL_JAVALANGILLEGALACCESSERROR;
 							*exceptionClass = methodClass;
+							*errorType = J9_VISIBILITY_NON_MODULE_ACCESS_ERROR;
```

Comment:
This is a really good find.  Since this is a bug fix, can it be split into a separate PR?

That PR can address the other exit points from this method and ensure they set errorType appropriately as well.

Reply:
Created a separate PR here: https://github.com/eclipse/openj9/pull/1383
Other exit points look like they handle it correctly; this was the odd one out.